### DISCUSSION TITLE
Remove It :|

### DISCUSSION THREAD

**Discussion by AliSh3ikhi** [projectdiscovery/nuclei#3135] [Ideas]
blah blah blah

**Comment by ehsandeep** [comment]
should be simple enough - `mv nuclei duplei`

**Reply by geeknik** [reply]
![image](https://user-images.githubusercontent.com/466878/210382078-a0ba1d17-9ed3-4a04-8138-305c978306ef.png)

The wonderful part of Open Source Software is that, if you following the License, you can fork nuclei into your own repository and change the name to anything you want. 👍🏻

**Reply by AliSh3ikhi** [reply]
It could be sarcasm
